src/aat/resources/features/F-103/S-598.td.json

```diff
@@ -0,0 +1,28 @@
+{
+  "title": "must return an error response for a malformed Case ID",
+  "_guid_": "S-598",
+  "_extends_": "F-103_Test_Data_Base",
+  "specs": [
+    "an active profile in CCD",
+    "contains an malformed case ID"
+  ],
+  "request": {
+    "pathVariables": {
+      "cid": "157375164884ab@!"
+    }
+  },
+  "expectedResponse": {
+    "_extends_": "Common_400_Response",
+    "headers": {
+      "Content-Encoding" : "[[ANYTHING_PRESENT]]",
```

Comment:
ANY_STRING_PRESENT is better.

Reply:
Fixed. Added the specific content-encoding, gzip.